packages/types/src/Space.ts

```diff
@@ -5,7 +5,7 @@
 import { HexString } from '@polkadot/util/types.js'
 import type { IPublicIdentity } from './PublicIdentity.js'
 
-export const SPACE_IDENTIFIER: number = 13
+export const SPACE_IDENTIFIER: number = 31
```

Comment:
We may not get a lot of options to change them once we start wide adoption. (well, we can 'add' to this list). We are good here? Or you expect any other changes in the number?